packages/video_player/video_player_android/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-## NEXT
+## 2.3.11+1
```

Comment:
@stuartmorgan The version check is complaining about this version, per your earlier comment:

```
This can just be '<some version>+<hotfix version>' since it's not changing any public API.
```

Reply:
That comment was on a package with a version number less than 1.0. See https://dart.dev/tools/pub/versioning#semantic-versions

Reply:
`avoid using + after the version reaches 1.0.0` Got it, and done.